src/utils/create-facttable.ts

```diff
@@ -0,0 +1,31 @@
+import { Database } from 'duckdb-async';
```

Comment:
missing hyphen in file name? `create-fact-table`?

or maybe `create-empty-cube` ?

Reply:
renamed to `create-fact-table`...  I suspect I might need a `create-empty-cube` at some point soon.  The difference being that one just creates tables and views with no data in them.  Whereas this creates a cube with just a populated fact table and none of the lookup tables.